src/resources/views/components/create-profile.blade.php

```diff
@@ -0,0 +1,80 @@
+@extends('layouts.app')
+
+@section('title', 'Create Profile')
+
+@section('content')
+    <div class="form-container">
+        <div class="title">
+            <h3 class="text-mywhite mix-blend-overlay">Profile Information</h3>
+        </div>
+
+        <form action="#" id="signup-form">
+            <div class="user-details">
+
+                <div class="input-box">
+                    <input type="text" name="first_name" placeholder="Juan" autocomplete="off" required>
+                    <span>First Name</span>
+                </div>
+
+                <div class="input-box">
+                    <input type="text" name="last_name" placeholder="Dela Cruz" autocomplete="off" required>
+                    <span>Last Name</span>
+                </div>
+
+                <div class="input-box">
+                    <input type="date" name="birth_date" id="bday" required>
+                    <span>Birthday</span>
+                </div>
+
+                <div class="input-box">
+                    <input type="text" name="contact" placeholder="09123456789" pattern="[0-9]{11}" autocomplete="off" required>
+                    <span>Phone Number</span>
+                </div>
+
+                <div class="user-details1">
+                    <div class="title">
+                        <h3 class="text-mywhite mix-blend-overlay">Address Information</h3>
+                    </div>
+                    <div class="input-box1">
+                        <input type="text" name=""  autocomplete="off" required>
+                        <span>Lot & Blk #</span>
+                    </div>
+
+                    <div class="input-box1">
+                        <input type="text" name="street"  autocomplete="off" required>
+                        <span>Street</span>
+                    </div>
+
+                    <div class="input-box1">
+                        <input type="text" name="city"  autocomplete="off" required>
+                        <span>City</span>
+                    </div>
+
+                    <div class="input-box1">
+                        <input type="text" name="province"  autocomplete="off" required>
+                        <span>Province</span>
+                    </div>
+
+                    <div class="input-box1">
+                        <input type="text" name="country" autocomplete="off" required>
+                        <span>Country</span>
+                    </div>
+
+                    <div class="input-box1">
+                        <input type="text" name="zip"  autocomplete="off" required>
```

Comment:
I think you are not recommended to use `required`. Instead, handle the validation in the from request.

https://laravel.com/docs/11.x/validation